CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 Information about contributing to the
 [kubernetes code repo](README.md) lives in the
-[kubernetes community repo](https://github.com/kubernetes/community)
+[kubernetes community repo](https://git.k8s.io/community/contributors/guide/)
 (it's a big topic).
```

Comment:
Since we're redoing the contributor guide, we can afford to be a bit more definite here. As well, explaining about joining the community and linking to k/community, e.g:

# Contributing

Welcome to Kubernetes! If you are interested in contributing to the [Kubernetes
code repo](README.md) then check out the  [Contributor
Guide](https://git.k8s.io/community/contributing)

The [Kubernetes community repo](https://github.com/kubernetes/community)
contains information on how the community is organized and other information
that is pertinent to contributing.

Reply:
@guineveresaenger  Thanks a lot for reviewing the PR, I did the changes :)